megatron/training.py

```diff
@@ -835,6 +838,7 @@ def backward_step(neox_args, timers, optimizer, model, loss):
 
 def train_step(neox_args, timers, data_iterator, model, optimizer, lr_scheduler):
     """Single training step."""
+    modules_dict = dict(model.named_modules())
```

Comment:
what's this for?

Reply:
oops, missed this in cleanup. @bclyang this was just for debugging via printing parameter values right?

Reply:
yes, thanks!